build_tools/__init__.py

```diff
@@ -32,4 +35,6 @@
 For BUILD_TIME_PLUGINS, RUN_TIME_PLUGINS, and EXTRA_REQUIREMENTS if the environment variable
 specifies a URL, the scripts will attempt to download a text file from that URL and then use
 that file.
+
+BACKGROUND_IMAGE, an image to display on the Kolibri login page.
```

Comment:
Should remove this, as it is specific to the UN women plugin.

Reply:
thanks! I will remove it!